### ISSUE TITLE
Print a ”standard” message when the nREPL server is started

### ISSUE THREAD

**Issue by PEZ** [borkdude/nbb#70]
**Is your feature request related to a problem? Please describe.**

In [adding **nbb Jack-in** to Calva ](https://github.com/BetterThanTomorrow/calva/pull/1312) the current message format when nbb has started its nREPL server tripped me a bit. It looks like so:

```
nRepl server started on port %d . nrepl-cljs-sci version %s 1337 TODO
```

So clearly it is just temporary and because the nREPL server is totally new.

**Describe the solution you'd like**

When babashka starts it prints a message like so:

```
Started nREPL server at 127.0.0.1:1667
```

Which Calva is prepared for and from where it picks up the host and port.

When the ”regular” Clojure nREPL server is started it prints:

```
nREPL server started on port 54836 on host localhost - nrepl://localhost:54836
```

Which Calva also is prepared for.

I know this is not really an API, but Calva certainly uses it as if it were. It would be nice if both nbb and babashka followed the nREPL pattern.

But since Calva already is adapted to babashka's format, I would be fine with nbb using that as well. I just wish that nbb will not invent some new format here.

**Comment by borkdude** [comment]
For nbb let's follow the nREPL pattern.

**Comment by borkdude** [comment]
Released in v0.0.90
